core/domain/exp_domain.py

```diff
@@ -286,7 +286,9 @@ class ExplorationChange(change_domain.BaseChange):
             'property_name', 'state_name', 'new_value'],
         'optional_attribute_names': ['old_value'],
         'user_id_attribute_names': [],
-        'allowed_values': {'property_name': STATE_PROPERTIES}
+        'allowed_values': {'property_name': STATE_PROPERTIES},
+        # TODO(#12991): Remove this once we migrate the data.
```

Comment:
Have the TODO be more specific -- which data? Developers may not have full context, so worth being explicit.

Ditto below and elsewhere.

Reply:
I have updated the todo comment.